Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PIDIntegralTime.mo

```diff
@@ -0,0 +1,100 @@
+﻿within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO;
+block PIDIntegralTime "Identifies the integral time of a PID controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput T(final min=0)
+    "Time constant of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput L(final min=1E-6)
+    "Time delay of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput Ti
+    "Time constant signal for the integral term"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Continuous.Add add1
+    "Calculate the sum of the time delay and the output of gai3"
+    annotation (Placement(transformation(extent={{-40,10},{-20,30}})));
+  Buildings.Controls.OBC.CDL.Continuous.Add add2
+    "Calculate the sum of the output of gai1 and the output of gai2"
+    annotation (Placement(transformation(extent={{-40,-30},{-20,-10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Divide div
```

Comment:
This block icon in diagram layer is upside down, so it looks like the `u2/u1`. Please flip it back.

Reply:
done